### PR TITLE
opensnitch, opensnitch-ui: 1.4.3 -> 1.5.0

### PR DESCRIPTION
**Pull request by onny**
###### Motivation for this change
Update Opensnitch to version 1.5.0

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/networking/opensnitch/ui.nix [hunk 1]
```diff
@@ -48,6 +50,8 @@ python3Packages.buildPythonApplication rec {
   dontWrapQtApps = true;
   makeWrapperArgs = [ "\${qtWrapperArgs[@]}" ];
 
+  doCheck = false; # checks fail for now
```
Comment (jonringer) [inline pkgs/tools/networking/opensnitch/ui.nix:53]: it would be nice for sandbox incompatible tests to be disabled. But turning off all tests isn't great.

Comment (reedriley) [reply pkgs/tools/networking/opensnitch/ui.nix:53]: I think _all_ the tests for opensnitch-ui are currently sandbox-incompatible.  At least; that was the impression I got from their failures when I ran them outside an X session by hand.

**Review by reedriley — APPROVED** [review]
This is basically identical to my pull request https://github.com/NixOS/nixpkgs/pull/158164; which I built and tested and works on my machine.

I'd note - I had huge functionality issues with opensnitch 1.4.3 and wireguard; which seem to be resolved in 1.5.0.  So strongly support upgrading ASAP.

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 158107 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>2 packages built:</summary>  <ul>    <li>opensnitch</li>    <li>opensnitch-ui</li>  </ul></details>

**Pull request merged**